src/CppParser/Parser.cpp

```diff
@@ -63,6 +63,25 @@ using namespace CppSharp::CppParser;
 // We use this as a placeholder for pointer values that should be ignored.
 void* IgnorePtr = (void*) 0x1;
 
+bool IsNamespaceStd(const clang::DeclContext* Namespace)
```

Comment:
I got confused here, I thought it already was because it's not registered in the header, something like a C function. I'll change it.